### PR TITLE
Set default less and css to new default name

### PR DESCRIPTION
**Pull request by joubertredrat**
Remove mentions to Gogs on js, css and less files, generated with lessc 2.7.1

### CODE REVIEW

File: public/css/index.css [hunk 1]
```diff
@@ -31,23 +31,18 @@ pre.wrap,
 code.wrap {
```
Comment (tboerger) [inline public/css/index.css:31]: Unrelated changes for that PR?

**Review by tboerger — CHANGES_REQUESTED** [review]

File: public/css/index.css (hunk 1 above)
Comment (joubertredrat) [reply public/css/index.css:31]: Only unnecessary spaces was removed. selector continue with same definitions.
https://github.com/go-gitea/gitea/blob/master/public/css/gogs.css#L31
https://github.com/joubertredrat/gitea/blob/changes-less-css/public/css/index.css#L31

Comment (tboerger) [reply public/css/index.css:31]: But there are css changes, you just renamed the less file.

Comment (joubertredrat) [reply public/css/index.css:31]: @tboerger was my fault, If you see on PR https://github.com/gogits/gogs/pull/3786, don't have empty lines on css, I don't know why, my npm display that less version is 2.7.1, but on my terminal was using 1.4.2. Now I solved this and generated correct css files, with no empty files and with `!important` as defined on less.

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM

**Comment by DblK** [comment]
LGTM, Just need a rebase

**Comment by joubertredrat** [comment]
Done, with rebase

**Comment by joubertredrat** [comment]
LGTM

**Comment by tboerger** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Comment by xinity** [comment]
LGTM

**Pull request merged**